internal/host/transition_test.go

```diff
@@ -38,6 +40,10 @@ func createValidatorCfg() *hardware.ValidatorCfg {
 	}
 }
 
+func createHardwareValidator(logrus logrus.FieldLogger) hardware.Validator {
+	return hardware.NewValidator(logrus, *createValidatorCfg())
```

Comment:
if it's a unit-test you should use the mock of this package. unit tests should test only the current package.